docs/theming.md

```diff
@@ -11,15 +11,68 @@ As of Jan 2017, skins are not fully supported; riot is the only available skin.
 To define a theme for Riot:
 
  1. Pick a name, e.g. `teal`. at time of writing we have `light` and `dark`.
- 2. Fork `src/skins/vector/css/themes/dark.scss` to be teal.scss
- 3. Fork `src/skins/vector/css/themes/_base.scss` to be _teal.scss
- 4. Override variables in _teal.scss as desired. You may wish to delete ones
-    which don't differ from _base.scss, to make it clear which are being
-    overridden. If every single colour is being changed (as per _dark.scss)
+ 2. Fork `src/skins/vector/css/themes/dark.scss` to be `teal.scss`
+ 3. Fork `src/skins/vector/css/themes/_base.scss` to be `_teal.scss`
+ 4. Override variables in `_teal.scss` as desired. You may wish to delete ones
+    which don't differ from `_base.scss`, to make it clear which are being
+    overridden. If every single colour is being changed (as per `_dark.scss`)
     then you might as well keep them all.
  5. Add the theme to the list of entrypoints in webpack.config.js
  6. Add the theme to the list of themes in matrix-react-sdk's UserSettings.js
  7. Sit back and admire your handywork.
 
 In future, the assets for a theme will probably be gathered together into a
 single directory tree.
+
+Custom Themes
+=============
+
+Themes derived from the built in themes may also be defined in settings.
+
+To avoid name collisions, the internal name of a theme is `custom-${theme.name}`. So if you want to set the custom theme below as the default theme, you would use `default_theme: "custom-Electric Blue"`.
```

Comment:
line wrap, but otherwise lgtm:
```suggestion
To avoid name collisions, the internal name of a theme is `custom-${theme.name}`.
So if you want to set the custom theme below as the default theme, you would use
`default_theme: "custom-Electric Blue"`.
```

Reply:
oh, I missed you'd done a suggestion and did it manually